src/ripples/focus-ripple.jsx

```diff
@@ -57,6 +80,8 @@ const FocusRipple = React.createClass({
   },
 
   _getRippleElement(props) {
+    const theme = this.state.muiTheme.ripple;
```

Comment:
Is that intentional, rather than the standard state method?  Does doing it this way here provide advantage/disadvantage over obtaining it from the context?